docs/SOAP-Service-Methods.md

```diff
@@ -582,6 +584,66 @@ Unblock activation with given ID. Activations can be unblocked in BLOCKED state
 | `String` | `activationId` | An identifier of an activation |
 | `ActivationStatus` | `activationStatus` | An activation status |
 
+### Method 'lookupActivations'
+
+Lookup activations using query parameters.
+
+#### Request
+
+`LookupActivationsRequest`
+
+| Type | Name | Description |
+|------|------|-------------|
+| `String` | `userIds` | User IDs to use in query, at least one user ID needs to be specified |
+| `String` | `applicationIds` | Application IDs to use in the query, do not specify value for all applications |
+| `String` | `timestampLastUsedBefore` | Filter activations by timestamp when the activation was last used (timestampLastUsed < timestampLastUsedBefore), do not specify value for any timestamp |
```

Comment:
The documentation related to timestamps is a little confusing - the formulation "do not specify value for any timestamp". For `timestampLastUsedBefore`, the docs should state "if not specified, a current timestamp is used". For `timestampLastUpdateAfter`, the docs should say "if not specified, the epoch start is used"

Reply:
Yes, that is much better.